src/utils.ts

```diff
@@ -347,6 +351,14 @@ export function txStatusToReceipt(txStatus: TransactionStatus): routes.Receipt {
       };
     });
 
+  let refundTxs = [];
+  if (txStatus.refundTxHash) {
+    refundTxs.push({
+      chain: srcChain,
```

Comment:
I think the refund chain depends on the method of transfer:

MCTP (and fast transfer swap layer) -> USDC on destination 
WH Swap and Swift -> input token on source chain

Do we need to consider that here?

Reply:
yeah you're right I was lazy here and assumed source chain because that was the only case I had seen in testing so far. but I just noticed the API response has `refundChain` in it, so will update this to read that.

Reply:
addressed in `a9b71db4f358384a300267493b72123875a04baa`